tests/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 if (${NOTEST})
     message(STATUS "Skipping test targets")
 else ()
+    add_subdirectory(unit-tests)
```

Comment:
What other tests are we expecting to necessitate the need for a subdirectory called "unit-tests"?